### PR TITLE
upload original points in addition to simplified

### PR DESCRIPTION
**Pull request by zaidalyafeai**
- [x] changed `processed_images` to `annotated_images`
- [x] upload original points in addition to the simplified ones 
- [x] fix the issue of single points

### CODE REVIEW

**Comment by zaidalyafeai** [comment]
@MagedSaeed there are two `calliar_images` folders in `media` and in `calliar_server` why is that?

File: calliar_server/server/static/utils.js [hunk 1]
```diff
@@ -38,4 +38,14 @@ function preprocess(name)
     }
 
     return [text, outText]
+}
+
+function getPoints(path){
+    var points = []
+    var segments = path.segments
+
+    for(i = 0; i< segments.length; i++){
```
Comment (MagedSaeed) [inline calliar_server/server/static/utils.js]: would it be better to use the `ForEach` syntax here?
```javascript
path.segments.forEach(segment => points.push([semgnet.point.x,segment.point.y])
```

**Comment by MagedSaeed** [comment]
@zaidalyafeai 

You are right. The one inside `media` foldr should be kept as it is the one mentioned in the `settings.py` file in `IMAGES_DIR` variable.

File: calliar_server/server/static/utils.js (hunk 1 above)
Comment (MagedSaeed) [reply calliar_server/server/static/utils.js]: This is an enhancement. It can be even done later. You can resolve this if you wish.

**Comment by MagedSaeed** [comment]
can you please remove the unnecessary folder in this PR?

File: calliar_server/server/static/utils.js (hunk 1 above)
Comment (zaidalyafeai) [reply calliar_server/server/static/utils.js]: fixed

**Comment by zaidalyafeai** [comment]
@MagedSaeed fixed all the issues.

**Pull request merged**